diagrams/request_uri_mode_create.md

```diff
@@ -0,0 +1,49 @@
+```plantuml
+@startuml
+
+autonumber
+
+box "Wallet"
+participant "Metadata" as wm
+participant "Authorization Endpoint" as w
+end box
+
+participant "User Agent" as u
+
+box "Verifier"
+participant "Frontend" as r
+participant "Request Endpoint" as rp
+participant "Response Endpoint" as rb
+end box
+
+u --> r : use
+activate r
+
+r --> u: authorization request\n(client_id, request{request_uri, request_uri_mode=POST, state})
```

Comment:
Is the intention of reusing the request_uri here? It will not work because according to this PR the request_uri is not a URI but a Request Object? Request_uri has to remain the same according to RFC9101 to have backwards-compatiblity. If this intending to be a breaking change then we should not overload request_uri but make the different paths more clear

Reply:
similar comment as above, right? See my answer there, pls.